src/DurableTask.Netherite/StorageLayer/Faster/AzureBlobs/BlobManager.cs

```diff
@@ -157,10 +174,21 @@ public class FasterTuningParameters
 
         public static string GetStorageFormat(NetheriteOrchestrationServiceSettings settings)
```

Comment:
I noticed this doesn't seem to be called in this PR. Is it called elsewhere?

Reply:
Yes, it is called when a new taskhub is created in `IStorageLayer.CreateTaskhubIfNotExistsAsync`. At that point, all the important parameters for this task hub storage format are recorded inside `taskhubparameters.json`.